### PR TITLE
vendor: update c/storage and c/common

### PR DESCRIPTION
**Pull request by giuseppe**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

/kind other
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by flouthoc** [comment]
Using following `.cirrus.yml` should omit all the other tests except `Containerized Integration` https://github.com/flouthoc/buildah/blob/do-not-merge/.cirrus.yml

**Comment by giuseppe** [comment]
@rhatdan @flouthoc PTAL

**Comment by giuseppe** [comment]
added a new patch to update c/image as well

File: contrib/cirrus/lib.sh [hunk 1]
```diff
@@ -181,10 +181,9 @@ in_podman() {
     done
     showrun podman run -i --name="$IN_PODMAN_NAME" \
                    --net="container:registry" \
-                   --security-opt=label=disable \
-                   --security-opt=seccomp=unconfined \
-                   --cap-add=all \
                    --env-file=$envfile \
+                   --privileged \
+                   --cgroupns=host \
```
Comment (flouthoc) [inline contrib/cirrus/lib.sh:186]: I think PR works without `--cgroupns=host` do we need this for any special reason.

**Review by flouthoc — APPROVED** [review]
LGTM
Just one question above.

File: contrib/cirrus/lib.sh (hunk 1 above)
Comment (giuseppe) [reply contrib/cirrus/lib.sh:186]: I think it is good to add it.  We are bind mounting `/sys/fs/cgroup` from the host, so the expectation is that the same cgroups are visible within the container

Comment (flouthoc) [reply contrib/cirrus/lib.sh:186]: ah I see, got it.

**Comment by flouthoc** [comment]
Needs a force merge since branch is outdated.

**Pull request merged**